Breadth_First_Search/Breadth_First_Search.rb

```diff
@@ -0,0 +1,92 @@
+=begin
```

Comment:
There is no need to add your personal github account link here. Anybody who sees this code in this repository can anyway see it is contributed by you. Please remove this begin end block